.github/workflows/build.yaml

```diff
@@ -2,6 +2,12 @@ name: Build
 on:
   schedule:
     - cron: "55 */6 * * *"
+  push:
+    branches:
+      - master
+    paths:
+      - 'build/*.js'
+      - 'config/*.json'
```

Comment:
Remember to add to the merge commit and/or the description that you changed CI

Reply:
i can do that! gimme a sec